hazelcast-spring/src/main/resources/hazelcast-spring-3.10.xsd

```diff
@@ -1834,7 +1834,7 @@
             </xs:documentation>
             <xs:appinfo>
```

Comment:
Probably we don't want to change this (Better to toss the old xsd's in a different PR)